vendor/bat-native-ads/src/bat/ads/internal/ml/data/vector_data.h

```diff
@@ -36,12 +36,18 @@ class VectorData final : public Data {
   VectorData& operator=(const VectorData& vector_data);
   VectorData& operator=(VectorData&& vector_data) noexcept;
 
+  // Mathematical vector operations
   friend double operator*(const VectorData& lhs, const VectorData& rhs);
-
+  void VectorAddElementWise(const VectorData& v_add);
```

Comment:
should we define new operators instead and return a `VectorData&`? I don't have a strong opinion on this, but one problem I see with this implementation is that the vector on which this is run is going to be overwritten with the sum/division result